griffier.py

```diff
@@ -100,12 +139,17 @@ def __init__(self, bot, host_id, utils):
 
 # Laad cogs
 bot.add_cog(CustomChannels(bot, utils))
-bot.add_cog(Aankondigingen(bot, utils))
-bot.add_cog(Groeter(bot, utils))
+# bot.add_cog(AutoRMTKAPI(bot, utils))
```

Comment:
Not very relevant, but this line slipped back into the branch after being removed.